packages/client/src/error.ts

```
@@ -130,7 +141,37 @@ export class ORPCError<TCode extends ORPCErrorCode, TData> extends Error {
+   *
+   * This is particularly problematic with "Optimized SSR", where orpc-client
+   * executes in one context but is invoked from another. When an error is thrown
+   * in the execution context, `instanceof ORPCError` checks fail in the
+   * invocation context due to separate class constructors.
+   *
+   * @todo Remove this and related code if Next.js resolves the multiple dependency graph issue.
+   */
+  static override[Symbol.hasInstance](instance: unknown): boolean {
+    // not applicable to extended classes
+    if (globalORPCErrorConstructors.has(this)) {
+      const constructor = getConstructor(instance)
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)